gapic/templates/%namespace/%name_%version/%sub/types/_enum.py.j2

```diff
@@ -4,6 +4,9 @@ class {{ enum.name }}({{ p }}.Enum):
     Values:
     {% for enum_value in enum.values %}
         {{ enum_value.name }} ({{ enum_value.number }}):
+        {% if enum_value.is_deprecated and 'deprecated' not in enum_value.meta.doc|lower %}
```

Comment:
The `and 'deprecated' not in enum_value.meta.doc|lower` bit is a little surprising.  Is it there just to avoid repeating the word "Deprecated" too many times?  But we don't know how "deprecated" might appear in `enum_value.meta.doc`.

Reply:
> Is it there just to avoid repeating the word "Deprecated" too many times? 
Correct. We are checking if the proto doc meets the AIP here: https://google.aip.dev/192#deprecations

I added a comment in [82c256c](https://github.com/googleapis/gapic-generator-python/pull/1738/commits/82c256c77950d7bdedd87bf40f8ed77135e02ee0)